token-metadata/Cargo.toml

```diff
@@ -2,4 +2,7 @@
 members=[
   "program",
   "cli"
-]
\ No newline at end of file
+]
+
+[profile.release]
+overflow-checks = true
```

Comment:
Not fully understanding why it has to be in this Cargo.toml (vs. inside the program), but maybe this is how our workspaces function?

Would hate to have this compile with this actually turned off and get deployed that way. Is it possible to add this in both places just to be sure?

Reply:
When you put it inside the `Cargo.toml` inside of `token-metadata/program` it gives you a warning that it will only be applied if it's in the workspace root `Cargo.toml` file. The matches [the docs](https://doc.rust-lang.org/cargo/reference/profiles.html).